src/Command/DocsCommand.php

```diff
@@ -26,7 +27,18 @@ protected function configure() {
    * @return int
    */
   protected function execute(InputInterface $input, OutputInterface $output) {
-    $this->localMachineHelper->startBrowser('https://docs.acquia.com/acquia-cli/');
+    $acquia_products = [
+      'Acquia ACLI' => 'https://docs.acquia.com/acquia-cli/',
+      'Acquia CMS' => 'https://docs.acquia.com/acquia-cms/',
```

Comment:
We probably should have a base URL value so we can change `docs.acquia.com` in one place should we need to.

Reply:
updated!